repository/query/query.go

```diff
@@ -33,6 +33,8 @@ var (
 	//go:embed scripts/update_project.sql
 	UpdateProject string
 
+	//go:embed scripts/read_invitation_by_email.sql
+	ReadInvitationByEmail string
```

Comment:
Naming will be updated once the conclusion is reached here - https://github.com/jan-zabloudil/release-manager/pull/69#pullrequestreview-2059661739